qiita_pet/templates/list_studies.html

```diff
@@ -0,0 +1,266 @@
+{% extends sitebase.html %}
+{% block head %}
+<link rel="stylesheet" href="/static/vendor/css/jquery.dataTables.css" type="text/css">
+
+<script src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<script src="/static/vendor/js/jquery.dataTables.plugin.natural.js"></script>
+
+<script type="text/javascript">
+var current_study;
+var ajaxURL = "/study/search/?&user={{current_user.id}}&sEcho=" + Math.floor(Math.random()*1001);
```

Comment:
Here you use  `*1001`, but later there are `sEcho` values of `1021` in your test code. Does this discrepancy matter? I admit, I don't fully understand the `sEcho` parameter here. Is it used to identify a particular request? If so, why do the requests need to be identified in this way? Lastly, if these are being used as identifiers, this does not allow for a tremendous number of simultaneous searches, why not increase 1001 to 1000001 or something?